03_ROS/deprecated/ghost_ros/CMakeLists.txt

```diff
@@ -51,24 +51,25 @@ include_directories(include)
 #####################################
 ##### Foxglove Diagnostics Node #####
 #####################################
```

Comment:
Not sure why these didn't sync from develop, but lets remove entirely. They are deprecated.